tests/transformers/test_transformers_model_export.py

```diff
@@ -2524,20 +2519,20 @@ def test_instructional_pipeline_with_prompt_in_output(model_path):
                 "hypothesis_template": "This example talks about how pipelines are {}",
             },
             {
-                "inputs": '[{"name": "sequences", "type": "string"}, {"name": '
-                '"candidate_labels", "type": "string"}, {"name": '
-                '"hypothesis_template", "type": "string"}]',
-                "outputs": '[{"name": "sequence", "type": "string"}, {"name": "labels", '
-                '"type": "string"}, {"name": "scores", "type": "double"}]',
```

Comment:
To confirm, this is all just no-op formatting, right? No fundamental reason for this particular key ordering?

Reply:
correct. I'm going to file a follow-up PR next week to convert all of these to dicts instead of JSON encoded dicts to minimize the chances of arbitrary ordering creating issues in these tests as well.